docs/DeveloperGuide.md

```diff
@@ -487,24 +503,35 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case ends.
 
-*{More to be added}*
 
 ### Non-Functional Requirements
 
-1.  Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
-1.  Should be able to hold up to 35000 students (NUS undergraduate cohort size) and 35000 appointments without a noticeable sluggishness in performance for typical usage.
-1.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
-1.  The product is not required to export statistics reports to PDF files.
-1.  There should be a mechanism to migrate data so that the application can be used with existing data on different machines.
+1.  **Accessibility**
+    * Vax@NUS should work on any _mainstream OS_ as long as it has Java `11` or above installed.
+1.  **Reliability**
+    * Vax@NUS should be able to hold up to 35000 students (NUS undergraduate cohort size) and 35000 appointments without a noticeable sluggishness in performance for typical usage.
+    * A user should be able to use Vax@NUS even while offline.
+1.  **Usability**
+    * A user with above average typing speed (41.4 words per minute) for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+    * A user should be able to use Vax@NUS without a mouse.
+1. **Security**
+    * A user should only be able to access Vax@NUS if they are an authorized UHC staff.
+    * The data in Vax@NUS should not be used or transferred to anywhere else unless authorized.
+1. **Quality**
+    * Vax@NUS should be able to be used by a novice who has never used a data management application.
+1. **Notes about project scope**
+    * Vax@NUS is not required to export statistics reports to PDF files.
+    * Vax@NUS is not required to export data for printing.
+
 
-*{More to be added}*
 
 ### Glossary
 
 * **Mainstream OS**: Windows, Linux, Unix, OS-X
+* **COVID-19**: The COVID-19 pandemic, also known as the coronavirus pandemic, is a global pandemic of coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2).
 * **Vaccine**: Singapore has two approved COVID-19 vaccines, one developed by Pfizer-BioNTech, 
   the other by Moderna. Both require two doses, 21 days apart for Pfizer-BioNTech, 28 days apart for Moderna.
-  Two appointments must therefore be created per student. 
+  Two appointments must therefore be created manually per student.
```

Comment:
But now we only allow 1 student 1 appointment. Shall we remove this line?

Reply:
Okay! I will remove it.